parl/remote/master.py

```diff
@@ -37,10 +37,11 @@ class Master(object):
 
     .. code-block:: python
 
-        xparl start --port localhost:1234
+        xparl start --port localhost:1234 --monitor_port 5005
```

Comment:
We should not mention monitor_port here. The usage here should be as compact as possible. Please add monitor_port at our documentation.